akka-docs/src/main/paradox/stream/operators/Flow/completionStageFlow.md

```diff
@@ -0,0 +1,42 @@
+# Flow.completionStageFlow
+
+Streams the elements through the given future flow once it successfully completes.
+
+@ref[Simple operators](../index.md#simple-operators)
+
+@@@div { .group-scala }
+
+## Signature
+
+@@signature [Flow.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala) { #futureFlow }
+
+@@@
+
+## Description
+
+Streams the elements through the given flow once the `CompletionStage` successfully completes. 
+If the future fails the stream is failed.
+
+## Examples
+
+A deferred creation of the stream based on the initial element like how the deprecated `lazyInit` worked can be achieved by combining `completionStageFlow`
```

Comment:
I don't think we should reference deprecated things from here. It will be obsolete. Mention in migration guide instead, and explain this as if the old didn't exist.

Reply:
Moved/updated